src/assets/components/Articles.jsx

```diff
@@ -15,11 +16,13 @@ const Articles  = () => {
         })
     }, [])
 
+    {isLoading ? (
```

Comment:
this isn't displaying anywhere that I can see, also what is the false side of your ternary doing?